src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $smoothly-duration;
```

```suggestion
    transition: color $smoothly-duration;
```
add properties for transition
default all, but lets write them directly - it can improve performance